src/components/Sidebar/Sidebar.tsx

```diff
@@ -40,7 +39,7 @@ export const ModalSidebar = styled.div`
   width: 340px;
 `
 
-export const StyledModalMain = styled(ModalMain)`
+export const StyledModalMain = styled.div`
```

Comment:
was there no benefits of using ModalMain in from styleguide?

Reply:
The entire `SimpleModal` was not used anywhere in the project, so I removed it.